Annulé : Mise en place de la documentation pour Github Pages

TristanEduProjet opened the pull request:
Mise en place de la documentation pour Github Pages
  - Ajout de la documentation sur les tests
  - Redirection du dossier de la documentation vers /docs

Mise en place de hooks pour automatiser la tâche.
Mise à jour de la documentation.

half-shell committed:
Add branches setup in the README #2

half-shell reviewed (commented):
T'as pas pull la merge request avant de faire cette pull-request :/

TristanEduProjet commented:
**Ne pas merger cette PullRequest pour l'instant !**

TristanEduProjet committed:
Mise en place de la documentation pour Github Pages
  - Ajout de la documentation sur les tests
  - Redirection du dossier de la documentation vers /docs
Mise en place de hooks pour automatiser la tâche.
Mise à jour de la documentation.

TristanEduProjet commented:
@half-shell tu peux vérifier si le pré-commit fonctionne sur Linux ?

half-shell commented:
@TristanEduProjet J'ai essayé, sans succès.
Le symllink est bien créé, une fois le `install.sh` lancé, mais quand je commit, il ne se passe rien (en dehors du commit). Je peux explorer un peu si ça marche de ton côté et que tu n'as pas d'idée d'où ça peut venir.

D'ailleurs, petite précision, mais il n'ya pas besoin d'être **root** pour lancer ce script, aucune des commandes ne le nécessite.

half-shell reviewed (changes requested):
Pour le script `install.sh` c'est mieux de mettre les chemins en absolus plutôt qu'en relatif.
Tu peux choper le root du projet en faisant
```bash
ROOT_DIR=$(git rev-parse --showtoplevel
# et préfixer les chemins avec
ln -s $ROOT_DIR/scripts/pre-commit.sh $ROOT_DIR/.git/hooks/pre-commit
```

TristanEduProjet commented:
**_Branches renommée_**
avec prise en compte de la nouvelle architecture ( #6 ) : #8
